### PR TITLE
Updating runc man page

### PR DESCRIPTION
**Pull request by rajasec**
Update man page for runc commands
Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

**Comment by cyphar** [comment]
LGTM

File: man/runc.8.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ value for "bundle" is the current directory.
    resume       resumes all processes that have been previously paused
    spec         create a new specification file
    start        create and run a container
+   update       update container resource constraints 
+   ps           ps displays the processes running inside a container
```
Comment (cyphar) [inline man/runc.8.md]: remove the leading `ps` from `ps displays`.

**Comment by rajasec** [comment]
@cyphar 
Updated.

**Comment by cyphar** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**